src/is/types.js

```diff
@@ -125,6 +125,13 @@ export function isWindowObject(value) {
     return value != null && typeof value === 'object' && 'setInterval' in value;
 }
 
+/**
```

Comment:
This should not be done in this file, but `environment.js` as mentioned in the issue.
Check out @boristane pull request #40